src/window.rs

```diff
@@ -190,6 +190,11 @@ pub struct WindowAttributes {
   /// The default is `false`.
   pub always_on_top: bool,
 
+  /// Whether the window should always be on bottom of other windows.
+  ///
+  /// The default is `false`.
+  pub always_below_bottom: bool,
```

Comment:
Should be renamed to 
```suggestion
  pub always_on_bottom: bool,
```
and all related methods should be renamed too

Reply:
3f4feba